traffic_ops/testing/api/v4/serverservercapability_test.go

```diff
@@ -222,6 +223,61 @@ func GetTestServerServerCapabilities(t *testing.T) {
 	}
 }
 
+func UpdateTestServerServerCapabilities(t *testing.T) {
+	var header http.Header
+
+	// Get server capability name and edit it to a new name
+	resp, _, err := TOSession.GetServerCapabilitiesWithHdr(header)
+	if err != nil {
+		t.Fatalf("Expected no error, but got %v", err.Error())
+	}
+	if len(resp) == 0 {
+		t.Fatal("no server capability in response, quitting")
+	}
+	origName := resp[0].Name
+	newSCName := "sc-test"
+	resp[0].Name = newSCName
+
+	// Get all servers related to original sever capability name
+	servResp, _, err := TOSession.GetServerServerCapabilitiesWithHdr(nil, nil, &origName, nil)
+	if err != nil {
+		t.Fatalf("cannot GET server capabilities assigned to servers by server capability name %v: %v", origName, err)
+	}
+	if len(servResp) == 0 {
+		t.Fatalf("no servers with associate server capability name: %s", origName)
+	}
+
+	// Update server capability with new name
+	updateResponse, _, err := TOSession.UpdateServerCapabilityByName(origName, &resp[0])
+	if err != nil {
+		t.Errorf("cannot PUT server capability: %v - %v", err, updateResponse)
+	}
+
+	//To check whether the primary key change trickled down to server table
+	ssc, _, err := TOSession.GetServerServerCapabilitiesWithHdr(nil, nil, &newSCName, nil)
+	if err != nil {
+		t.Fatalf("cannot GET server capabilities assigned to servers by server capability name %v: %v", newSCName, err)
+	}
+	if len(ssc) == 0 {
+		t.Fatalf("no server with associated server capability name:%s", newSCName)
+	}
+	for i, s := range ssc {
+		if newSCName != *ssc[i].ServerCapability {
```

Comment:
nit: I would suggest using the same format in both the checks (this one and the one on line 268)
Basically, either use `ssc[i]` or `s` in both the checks, because they are both the same element. Makes it easier for the reader to see whats going on.